tools/ssh_helpers/airlock.sh

```diff
@@ -3,16 +3,53 @@
 environment=${1}
 
 airlock_interface="ens4"
+ns_dir="/hab/svc/builder-worker/data/network/airlock-ns"
 
 for worker in $(grep "Host ${environment}-builder-worker" ~/.ssh/config | awk '{print $2}' | sort); do
     echo "Worker ${worker}"
-    output=$(ssh ${worker} "ip address show dev ${airlock_interface} | grep 'inet '")
-    if [ -z "$output" ]; then
-        echo "OMG NO IP! Restarting network stack..."
-        ssh ${worker} "sudo /etc/init.d/networking restart"
+    if ssh "$worker" sudo test -d "${ns_dir}"; then
+      # The Airlock network namespace is created
+      output="$(ssh "$worker" sudo nsenter --user="$ns_dir/userns" --net="$ns_dir/netns" ip address show dev "$airlock_interface" | grep 'inet ')"
+      if [ -n "$output" ]; then
+          # The Airlock interface is present in the network namespace--which is
+          # what we are expecting
+          echo "Network interface $airlock_interface in network namespace $ns_dir:"
+          echo "${output}"
+          echo "${worker} ${airlock_interface} => OK"
+      else
+          echo ">>> The Airlock interface isn't in the network namespace $ns_dir."
+          echo ">>>"
+          echo ">>> This is an issue."
+          echo ">>>"
+          echo ">>> Consider removing the Airlock network namespace with:"
```

Comment:
can we just issue these fix commands via the script - maybe if we specify a 'auto-fix' flag?